tf_encrypted/protocol/pond/pond.py

```diff
@@ -65,7 +67,7 @@ def __init__(
         self.server_1 = server_1 or get_config().get_player("server1")
         crypto_producer = crypto_producer or get_config().get_player("server2")
         crypto_producer = crypto_producer or get_config().get_player("crypto-producer")
-        self.crypto_producer = crypto_producer
+        self.triple_source = OnlineTripleSource(crypto_producer)
```

Comment:
clarifying that default behavior is Online Triple Generation in the docstring above might be helpful to our crypto friends